index.rst

```diff
@@ -1,70 +1,87 @@
+What can we help you with today?
 
-Clouds
-------
+.. raw:: html
 
-We offer multiple cloud options because we believe that everyone should have access to great data platforms to host their applications. Our customers tell us they love the multi-cloud approach because they know that they aren't locked in to one particular cloud platform for all their data needs.
+    <form class="sidebar-search-container" method="get" action="search.html">
+      <input class="sidebar-search" placeholder=Search name="q">
+      <input type="hidden" name="check_keywords" value="yes">
+      <input type="hidden" name="area" value="default">
+    </form>
 
-You can choose from Amazon Web Services (AWS), Google Cloud Platform (GCP), Microsoft Azure Cloud, DigitalOcean Cloud and UpCloud, and then pick the region where your service should be hosted.
 
 
-Pricing
--------
+(Looking for the previous documentation? Visit `Aiven Help <https://help.aiven.io>`_ or use the search above.)
 
-You are billed on a monthly basis, and you only pay for the time that the service is running. Data transfer costs are included. 
-You can check more on our `Pricing page <https://aiven.io/pricing>`_.
+----------------
 
-Products
---------
+Explore
+-------
 
-Relational databases that you know and love ❤️
+Learn about the Aiven platform
 
 .. panels::
+    :card: shadow
+
+    |icon-postgres| **PostgreSQL** Powerful relational database platform. We have the latest versions, and all the extensions you need.
 
-    |icon-postgres| **PostgreSQL** The lastest versions, and all the extensions you need.
+    +++
+
+    .. link-button:: docs/products/postgresql/index
+        :type: ref
+        :text: Read more
+        :classes: stretched-link
 
     ---
 
-    |icon-mysql| **MySQL** World's most widely-used open source database.
+    |icon-m3db| **M3DB** Distributed time-series database for scalable solutions, with M3 Coordinator included, and M3 Aggregator also available.
 
-Streaming data and events 📨
+    +++
 
-.. panels::
+    .. link-button:: docs/products/m3db/index
+        :type: ref
+        :text: Read more
+        :classes: stretched-link
 
-    |icon-kafka| **Apache Kafka** For data streaming services, look no further than Apache Kafka.
+    ---
+
+    |icon-kafka| **Apache Kafka** Streaming data pipelines for the modern enterprise. Apache MirrorMaker2 and Kafka Connect also available.
 
     ---
 
-    |icon-kafka-connect| **Kafka Connect** To connect all the things into (source) and out of (sink) Apache Kafka.
+    |icon-elasticsearch| **ElasticSearch** Document database with specialist search features, bring your freeform ddocuments, logs or metrics, and make sense of them here.
 
     ---
 
-    |icon-kafka-mirrormaker| **MirrorMaker2** Everything from one Apache Kafka to another.
+    |icon-cassandra| **Cassandra** High performance storage solution for large data quantities. This specialist data solution is a partitioned row store.
```

Comment:
"partitioned row store" I believe could be confusing? should we add NoSQL somewhere?

Reply:
it isn't NoSQL, and its own website describes it that way. I thought it was a columnar database but apparently not!